js/script.js

```diff
@@ -4,6 +4,7 @@ var source_of_truth = {
   turf_area: '',
   home_address: '',
   customCalscapeUrl: 'https://calscape.org/loc-/?&poploc=1',
+  customrebateUrl: 'https://www.google.com/search?q=water+efficency+rebate+programs&oq=water+efficency+rebate+programs&aqs=chrome..69i57.6904j0j7&sourceid=chrome&ie=UTF-8',
```

Comment:
Probably don't need the chrome parameters?

If no address is specified in the search bar maybe use "water efficiency programs near me" instead? This seems to work decently.

Reply:
Updated